index.html

```diff
@@ -24,8 +24,49 @@ <h2><input type="text" class="nes-input" placeholder="what needs to be done" v-o
     </li>
   </ul>
 
-  <center><button type="button">Active</button>
-  <button type="button">Completed</button></center>
+  <ul v-if="showActive"
+    <li v-for="todo in activeTodos" :key="todo.id" class="flex">
+      <label>
+        <input type="checkbox" class="nes-checkbox" v-model="todo.done">
+        <span>&nbsp</span>
+      </label>
+      <del v-if="todo.done">{{ todo.text }}</del>
+      <span v-else>{{ todo.text }}</span>
+      <div class="space"></div>
+      <button class="nes-btn is-error padding" v-on:click="removeTodo(todo.id)">X</button>
+    </li>
+  </ul>
+
+  <ul v-if="showCompleted">
+      <li v-for="todo in completedTodos" :key="todo.id" class="flex">
+      <label>
+        <input type="checkbox" class="nes-checkbox" v-model="todo.done">
+        <span>&nbsp</span>
+      </label>
+      <del v-if="todo.done">{{ todo.text }}</del>
+      <span v-else>{{ todo.text }}</span>
+      <div class="space"></div>
+      <button class="nes-btn is-error padding" v-on:click="removeTodo(todo.id)">X</button>
+    </li>
+  </ul>
+
+  <ul v-if="showAll">

```

Comment:
remove this list. Just add `<ul v-if="showAll">` to the original list here: https://github.com/xiomaraquinonez/todolist/pull/1/files#diff-eacf331f0ffc35d4b482f1d15a887d3bL14

Reply:
Add `<ul v-if="showAll">` to the original list on line 14 in index.html